elasticsearch/src/test/java/org/apache/calcite/adapter/elasticsearch/ElasticSearchAdapterTest.java

```diff
@@ -476,8 +484,8 @@ public static void setupInstance() throws Exception {
 
   @Test void testInPlan() {
     final String[] searches = {
-        "query: {'constant_score':{filter:{bool:{should:"
```

Comment:
Why did `bool:should` disappeared?

Reply:
After CALCITE-4606, Search RexCall will be translated to Terms Query, you can see the full explanation here: https://github.com/apache/calcite/pull/2420/files#r647115126.

Reply:
OK, got it!